src/Sarif/Autogenerated/Address.cs

```diff
@@ -42,7 +43,12 @@ public SarifNodeKind SarifNodeKind
         [DataMember(Name = "absoluteAddress", IsRequired = false, EmitDefaultValue = false)]
         [DefaultValue(-1)]
```

Comment:
>[DefaultValue(-1)] [](http://example.com/codeflow?start=8&length=18)

If DefaultValue doesn't work for BigInteger, should this be removed?

Reply:
tested, no it is needed when a json string that does not have the AbsoluteAddress node at all, to be able to construct an c# object that has AbsoluteAddress property as -1